.github/workflows/journey-map.yml

```diff
@@ -0,0 +1,47 @@
+# Simple workflow for deploying static content to GitHub Pages
+name: Deploy static content to Pages
+
+on:
+  # Runs on pushes targeting the default branch
+  push:
+    branches:
+      - main
+    paths:
+      - 'journey-map/**'
+      - 'lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/**'
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# Sets permissions of the GITHUB_TOKEN to allow deployment to GitHub Pages
+permissions:
+  contents: read
+  pages: write
+  id-token: write
+
+# Allow only one concurrent deployment, skipping runs queued between the run in-progress and latest queued.
+# However, do NOT cancel in-progress runs as we want to allow these production deployments to complete.
+concurrency:
+  group: "pages"
+  cancel-in-progress: false
```

Comment:
By defining the concurrency at the workflow level, does every run belong to the "pages" group by default? I was sort of expecting to see this block defined, and then the job reference that group in some way. The docs don't suggest that's how it works though.

Reply:
Ah gotcha. That makes sense now - I hadn't considered the concurrency of separate workflows.